app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.java

```diff
@@ -69,6 +69,14 @@ public static UrlInputFragment createWithBackground() {
         return fragment;
     }
 
+    @Override
+    public void applyLocale() {
+        getActivity().getSupportFragmentManager()
+                .beginTransaction()
+                .replace(R.id.container, UrlInputFragment.createWithBackground(), UrlInputFragment.FRAGMENT_TAG)
```

Comment:
I wonder if "withBackground" is always correct here? What happens if we show this as an overlay and I leave the app - go to the system settings, change the locale and resume the app?